modules/base/src/components/options/Option.vue

```diff
@@ -0,0 +1,350 @@
+          <span style="margin-left: 20px;"></span><strong>TELEM</strong>: Telemetry<br />
+          <span style="margin-left: 20px;"></span>Services that collect application telemetry.<br />
+          <span style="margin-left: 20px;"></span><strong>TRACO</strong>: Other<br />
+          <span style="margin-left: 20px;"></span>Services that are believed to serve ads or track use but their exact
+          use is unknown or not categorized.<br />
+          <strong>MAL</strong>: Malware<br />
+          Services that are (ab)used for attacking devices through technical means.<br />
+          <strong>DECEP</strong>: Deception<br />
+          Services that trick humans into thinking the service is genuine, while it is not. There is no malware
+          involved.<br />
+          <span style="margin-left: 20px;"></span><strong>PHISH</strong>: Phishing<br />
+          <span style="margin-left: 20px;"></span>Services that engage in password fishing.<br />
+          <span style="margin-left: 20px;"></span><strong>FAKEN</strong>: Fake News<br />
+          <span style="margin-left: 20px;"></span>Services that deliberately provide wrong information.<br />
+          <span style="margin-left: 20px;"></span><strong>FRAUD</strong>: Fraud<br />
+          <span style="margin-left: 20px;"></span>Services that scam people.<br />
+          <strong>BAD</strong>: Bad Stuff (Mixed)<br />
+          Miscellaneous services that are believed to be harmful to security or privacy, but their exact use is unknown,
+          not categorized, or lists <br />have mixed categories. See individual descriptions for more information.<br />
+          <strong>NSFW</strong>: NSFW<br />
+          Services that are generally not accepted in work environments.<br />
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import OptionString from "./OptionString.vue";
+import OptionStringSelect from "./OptionStringSelect.vue";
+import OptionStringArray from "./OptionStringArray.vue";
+import OptionStringArrayCountries from "./OptionStringArrayCountries.vue";
+import OptionBoolean from "./OptionBoolean.vue";
+import OptionIntSecurityLevel from "./OptionIntSecurityLevel.vue";
+import OptionInt from "./OptionInt.vue";
+
+export default {
+  name: "Option",
+  components: {
+    OptionString,
+    OptionStringSelect,
+    OptionStringArray,
+    OptionStringArrayCountries,
+    OptionBoolean,
+    OptionIntSecurityLevel,
+    OptionInt
+  },
+  props: {
+    rKey: String,
+    record: Object
+  },
+  data() {
+    return {
+      newValue: this.record.Value,
+      validationError: "",
+      request: {},
+      optTypeString: 1,
+      optTypeStringArray: 2,
+      optTypeInt: 3,
+      optTypeBool: 4,
+      showHelp: false
+    };
+  },
+  computed: {
+    successState() {
+      if (this.request.success === true) {
+        return true;
+      }
+      return false;
```

Comment:
```suggestion
return this.request.success;
```

or if the type safty is required,
```
return this.request.success === true;
```